web/src/utils/columns.ts

```diff
@@ -168,11 +204,15 @@ export const getDefaultColumns = (t: TFunction): Column[] => {
       id: ColumnsId.packets,
       name: t('Packets'),
       isSelected: true,
-      defaultOrder: 12,
+      defaultOrder: 0,
       filterType: FilterType.NONE,
       value: f => f.fields.Packets,
       sort: (a, b, col) => compareNumbers(col.value(a) as number, col.value(b) as number),
       width: 10
     }
   ];
+  cols.forEach((c, idx) => {
+    c.defaultOrder = idx + 1;
+  });
```

Comment:
Seems I forgot to remove `defaultOrder` field after fixing reset default button in popup :

```
  const onReset = React.useCallback(() => {
    setUpdatedColumns(getDefaultColumns(t));
  }, [setUpdatedColumns, t]);
```

Since we call `getDefaultColumns` again, we can remove `defaultOrder` field and this `forEach` loop